controllers/clusterversion_controller.go

```diff
@@ -436,3 +465,41 @@ func (c *ClusterVersionReconciler) ensureConsolePlugin() error {
 
 	return nil
 }
+
+func (c *ClusterVersionReconciler) reconcileCSIAddonsSubscription() error {
+	csiAddonsSubscription, err := c.getSubscriptionByPackageName(csiAddonsSubscriptionPackageName)
+	if err != nil {
+		return err
+
+	}
+
+	//the channel of csiAddons subscription here is stable-4.15 because of a direct merge to release-4.15
+	//this code shouldn't be back ported and from release-4.16 the logic is changed
```

Comment:
```suggestion
// The channel of csiAddons subscription here is stable-4.15 because of a direct merge to release-4.15. This code shouldn't be backported, and from release-4.16, the logic has changed.
```

add a space between `//` and the word

Reply:
done